HHS/Head-Start-TTADP
# Use csv-parse on csv front end import

## Description of change

Tammy discovered a limitation with the front end code we use to verify a valid csv (before the BE gets it). This ticket changes the front end to verify the csv file with the csv-parse node module. This takes the guess work out strings that contain comma's.

**Note:** Will need to run docker deps to pull FE node module.

## How to test

Make sure you can import courses that contain a comma and some that do not.
Ex1: "My sample, course 1"
Ex2: "My sample course 2

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-0


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
